Post_request.py

```diff
@@ -1,9 +1,13 @@
 import requests
+import json
+
 # local url
-url = 'http://127.0.0.1:5000/api/summarize'   # change to your url
-curl --request POST \
-    --url http://127.0.0.1:5000/api/summarize \
-    --header 'content-type: application/json' \
-    --data '{
-        "articleUrl": "     "
-    }'
\ No newline at end of file
+url = 'http://127.0.0.1:5000/api/summarize'
+
+article = {'Data': Your article goes here}
+
+article = json.dumps(article)
+
+send_request = requests.post(url, article)
```

Comment:
Firstly, I created a variable with the API endpoint then provided a dictionary for loading the article.
Lastly, the article was serialized into a json object by invoking json.dumps() and the post request to the API endpoint was created.